applications/visualizer/frontend/src/components/viewers/EM/EMStackTilesViewer.tsx

```diff
@@ -117,6 +129,20 @@ function onNeuronSelect(position: Coordinate, source: VectorSource<Feature> | un
   workspace.addSelection(neuronName, ViewerType.EM);
 }
 
+function onSynapseSelect(feature: Feature, _: Workspace) {
+  const synapseName = neuronFeatureName(feature);
+
+  const selected = feature.get("selected");
```

Comment:
It would be better to not add special attributes on features, but to use the synchronizers instead, otherwise, it will be complicated to keep this in sync later.